spring/boot3-webflux-autoconfigure/build.gradle

```diff
@@ -0,0 +1,67 @@
+configurations {
+    // TODO(hyangtack) The followings are not transitive dependencies and they are not in testRuntime
+    //                 dependencies. Need to figure out why they are included in shadedTestRuntime dependencies.
+    // Exclude jetty from shadedTest.
+    shadedTestRuntime.exclude group: 'org.eclipse.jetty'
+    shadedTestRuntime.exclude group: 'org.eclipse.jetty.http2'
+}
+
+dependencies {
+    // To let a user choose between thrift and thrift0.9.
+    compileOnly project(':thrift0.17')
+    implementation project(':logback')
+
+    api libs.spring.boot3.starter.webflux
+    api libs.jakarta.inject
+    compileOnly libs.jakarta.validation
+    // We may remove jakarta.websocket dependencies if Armeria webflux module supports WebSocket.
+    implementation libs.jakarta.websocket
```

Comment:
I'm not sure if we need this or not. Shouldn't the user who wants to use WebSocket include this dependency?

Reply:
https://github.com/spring-projects/spring-framework/blob/main/spring-webflux/src/main/java/org/springframework/web/reactive/socket/server/support/HandshakeWebSocketService.java#L272-L291
WebFlux in Spring Boot 3 uses Jakarta Websocket as a fallback if no custom WebSocket `RequestUpgradeStrategy`. The explanation was not enough. Let me update the comment.

Reply:
Thanks a lot for the explanation. 🙇